# Практическая работа №3: Обработка выборочных данных. Нахождение интервальных оценок параметров распределения. Проверка статистической гипотезы о нормальном распределении

Выполнили студенты гр. 2381 Ивашинников Леонид Дмитриевич и Двиков Даниил Владиславович. Вариант №23.

## Цель работы

Получение практических навыков вычисления интервальных статистических оценок параметров распределения выборочных данных и проверки «справедливости» статистических гипотез.

## Основные теоретические положения

**Интервальная оценка** — оценка параметра Θ в виде интервала (Θ* − δ; Θ* + δ), где δ — точность, γ = P{|Θ* − Θ| < δ} — надёжность (доверительная вероятность). Уровень значимости q = 1 − γ.

Доверительный интервал для мат. ожидания (σ неизвестно)

При неизвестном σ величина $t = \frac{\bar{x} - a}{S / \sqrt{N}}$ распределена по закону Стьюдента с $k = N - 1$ степенями свободы. Доверительный интервал:

$$\left(\bar{x} - \frac{t_\gamma S}{\sqrt{N}};\ \bar{x} + \frac{t_\gamma S}{\sqrt{N}}\right)$$

где $t_\gamma$ определяется по таблице распределения Стьюдента (Приложение 6).

Доверительный интервал для СКО

Представив $\delta = qS$, доверительный интервал для σ:

$$(S(1 - q);\ S(1 + q))$$

где $q$ определяется по таблице (Приложение 7) для заданных γ и N.

Критерий Пирсона χ²

Гипотеза $H_0$: выборочные данные — значения нормально распределённой СВ. Наблюдаемое значение (правосторонний критерий):

$$\chi^2_{\text{набл}} = \sum_{i=1}^{K} \frac{(n_i - n'_i)^2}{n'_i}$$

Вычисление теоретических частот: $z_i = (x_i - \bar{x}) / S$ ($z_1 = -\infty$, $z_{K+1} = +\infty$), $p_i = \Phi(z_{i+1}) - \Phi(z_i)$ (Приложение 4), $n'_i = N \cdot p_i$.

Число степеней свободы: $k = K - 3$. Критическое значение $\chi^2_{\text{крит}} = \chi^2(\alpha, k)$ (Приложение 5).

Область принятия $H_0$: $\chi^2_{\text{набл}} \leq \chi^2_{\text{крит}}$.

Ошибки при проверке гипотез

Ошибка I рода (α) — отвергнуть верную $H_0$. Ошибка II рода (β) — принять неверную $H_0$. Мощность критерия — $1 - \beta$.

## Постановка задачи

Для заданной надежности определить (на основании выборочных данных и результатов выполнения практической работы №2) границы доверительных интервалов для математического ожидания и среднеквадратичного отклонения случайной величины. Проверить гипотезу о нормальном распределении исследуемой случайной величины с помощью критерия Пирсона $\chi^2$. Дать содержательную интерпретацию полученным результатам.

## Порядок выполнения работы

1. Вычислить точность и доверительный интервал для математического ожидания при неизвестном среднеквадратичном отклонении при заданном объёме выборки для доверительной точности $\gamma \in \{0.95, 0.99\}$. Сделать выводы.
2. Для вычисления границ доверительного интервала для среднеквадратичного отклонения определить значение $q$ при заданных $\gamma$ и $n$. Построить доверительные интервалы, сделать выводы.
3. Проверить гипотезу о нормальности заданного распределения с помощью критерия $\chi^2$ (Пирсона). Для этого необходимо найти теоретические частоты и вычислить наблюдаемое значение критерия. Для удобства вычисления необходимо заполнить табл. 1.
4. Доказать, что
$$\chi^2_{\text{набл}} = \sum_{i=1}^{k} \frac{n_i^2}{n'_i} - n.$$
Проконтролировать корректность вычисления $\chi^2_{\text{набл}}$.

5. По заданному уровню значимости $\alpha = 0.05$ и числу степеней свободы $df$ найти критическую точку $\chi^2_{\text{крит}}$ и сравнить с наблюдаемым значением. Сделать выводы.

## Расчётные таблицы
### Таблица №1
| $i$    | $[x_i, x_{i+1})$ | $n_i$ | $p_i$ | $n'_i$ | $(n_i - n'_i)^2$ | $\cfrac{(n_i - n'_i)^2}{n'_i}$ | $n_i^2$ | $\cfrac{n_i^2}{n'_i}$ |
|------|-----------------|-----|------|-------|-----------------|-------------------------------|-------|---------------------|
| 1    |                 |     |      |       |                 |                               |       |                     |
| ...  |                 |     |      |       |                 |                               |       |                     |
| $\Sigma$ |            |     |      | -     |                 |                               | -     |                     |

## Выполнение работы

In [22]:
import pandas as pd
import numpy as np
from scipy import stats

df = pd.read_csv('dataset.csv', sep=';', decimal='.')
print(f'Столбцы: {df.columns.tolist()}')
df.describe()

Столбцы: ['nu (kg/cm^3)', 'E (kg/cm^2)']


,nu (kg/cm^3),E (kg/cm^2)
count,109.000000,109.000000
mean,0.449569,127.912844
std,0.054279,22.346406
min,0.320000,71.100000
25%,0.412000,114.800000
50%,0.448000,127.700000
75%,0.487000,143.800000
max,0.593000,187.400000


1. Вычислить точность и доверительный интервал для математического ожидания при неизвестном среднеквадратичном отклонении при заданном объёме выборки для доверительной точности $\gamma \in \{0.95, 0.99\}$. Сделать выводы.

In [23]:
nu = df['nu (kg/cm^3)']
E = df['E (kg/cm^2)']

N = len(df)
k = N - 1    # число степеней свободы для распределения Стьюдента

# Выборочное среднее
x_mean_nu = nu.mean()
x_mean_E = E.mean()

# Исправленное выборочное СКО
S_nu = nu.std(ddof=1)
S_E = E.std(ddof=1)

print(f'N = {N}')
print(f'k = N - 1 = {k}')
print(f'nu (кг/см³)')
print(f'Выборочное среднее x̄ = {x_mean_nu:.4f}')
print(f'Исправленное СКО   S = {S_nu:.4f}')
print(f'E (кг/см²)')
print(f'Выборочное среднее x̄ = {x_mean_E:.4f}')
print(f'Исправленное СКО   S = {S_E:.4f}')

N = 109
k = N - 1 = 108
nu (кг/см³)
Выборочное среднее x̄ = 0.4496
Исправленное СКО   S = 0.0543
E (кг/см²)
Выборочное среднее x̄ = 127.9128
Исправленное СКО   S = 22.3464


Нахождение квантилей распределения Стьюдента. Для двустороннего доверительного интервала с надёжностью γ квантиль $t_\gamma$ находится из условия:

$$P\left(|t| < t_\gamma\right) = \gamma \quad \Rightarrow \quad t_\gamma = t_{(1+\gamma)/2,\, k}$$

In [24]:
gammas = [0.95, 0.99]

print(f'Число степеней свободы: k = {k}')
print()

for gamma in gammas:
    # Квантиль распределения Стьюдента
    t_gamma = stats.t.ppf((1 + gamma) / 2, df=k)
    print(f'γ = {gamma}: t_γ = t({(1+gamma)/2}, {k}) = {t_gamma:.4f}')

Число степеней свободы: k = 108

γ = 0.95: t_γ = t(0.975, 108) = 1.9822
γ = 0.99: t_γ = t(0.995, 108) = 2.6221


Вычисление точности δ и доверительных интервалов

Точность оценки:
$$\delta = \frac{t_\gamma \cdot S}{\sqrt{N}}$$

Доверительный интервал:
$$\left(\bar{x} - \delta;\; \bar{x} + \delta\right)$$

In [25]:
# Функция для вычисления доверительного интервала при неизвестном СКО. 
def confidence_interval(x_mean, S, N, gamma):
    k = N - 1
    t_gamma = stats.t.ppf((1 + gamma) / 2, df=k)
    delta = t_gamma * S / np.sqrt(N)
    left = x_mean - delta
    right = x_mean + delta
    return t_gamma, delta, left, right


In [26]:
print(f'Параметр: nu (кг/см³)')
print(f'x̄ = {x_mean_nu:.4f}, S = {S_nu:.4f}, N = {N}, k = {k}')
print('=' * 65)

for gamma in gammas:
    t_g, delta, left, right = confidence_interval(x_mean_nu, S_nu, N, gamma)
    
    print(f'\nγ = {gamma}:')
    print(f'  t_γ = {t_g:.4f}')
    print(f'  δ = t_γ · S / √N = {t_g:.4f} · {S_nu:.4f} / √{N} = {delta:.4f}')
    print(f'  Доверительный интервал: ({left:.4f}; {right:.4f})')

print(f'Параметр: E (кг/см²)')
print(f'x̄ = {x_mean_E:.4f}, S = {S_E:.4f}, N = {N}, k = {k}')
print('=' * 65)

for gamma in gammas:
    t_g, delta, left, right = confidence_interval(x_mean_E, S_E, N, gamma)
    
    print(f'\nγ = {gamma}:')
    print(f'  t_γ = {t_g:.4f}')
    print(f'  δ = t_γ · S / √N = {t_g:.4f} · {S_E:.4f} / √{N} = {delta:.4f}')
    print(f'  Доверительный интервал: ({left:.4f}; {right:.4f})')

Параметр: nu (кг/см³)
x̄ = 0.4496, S = 0.0543, N = 109, k = 108

γ = 0.95:
  t_γ = 1.9822
  δ = t_γ · S / √N = 1.9822 · 0.0543 / √109 = 0.0103
  Доверительный интервал: (0.4393; 0.4599)

γ = 0.99:
  t_γ = 2.6221
  δ = t_γ · S / √N = 2.6221 · 0.0543 / √109 = 0.0136
  Доверительный интервал: (0.4359; 0.4632)
Параметр: E (кг/см²)
x̄ = 127.9128, S = 22.3464, N = 109, k = 108

γ = 0.95:
  t_γ = 1.9822
  δ = t_γ · S / √N = 1.9822 · 22.3464 / √109 = 4.2426
  Доверительный интервал: (123.6702; 132.1555)

γ = 0.99:
  t_γ = 2.6221
  δ = t_γ · S / √N = 2.6221 · 22.3464 / √109 = 5.6124
  Доверительный интервал: (122.3005; 133.5252)


Вывод: Доверительные интервалы для математического ожидания построены с использованием распределения Стьюдента ($k = N - 1 = 108$). Полученные значения $t_\gamma$ согласуются с табличными (Приложение 6).

При увеличении надёжности с $\gamma = 0.95$ до $\gamma = 0.99$ точность оценки снижается ($\delta$ возрастает), что приводит к расширению доверительного интервала. Это соответствует теоретическому положению: повышение доверительной вероятности возможно только за счёт увеличения ширины интервала при фиксированном объёме выборки.

2. Для вычисления границ доверительного интервала для среднеквадратичного отклонения определить значение $q$ при заданных $\gamma$ и $n$. Построить доверительные интервалы, сделать выводы.

In [27]:
# q из Приложения 7 
q_table = {0.95: 0.143, 0.99: 0.198}

print(f'Параметр: nu (кг/см³)')
print(f'S = {S_nu:.4f}, N = {N}, k = {k}')
print('=' * 65)

for gamma in gammas:
    q = q_table[gamma]
    left = S_nu * (1 - q)
    right = S_nu * (1 + q)

    print(f'\nγ = {gamma}:')
    print(f'  q = {q} ')
    print(f'  Доверительный интервал: (S(1-q); S(1+q)) = ({left:.4f}; {right:.4f})')

print(f'\nПараметр: E (кг/см²)')
print(f'S = {S_E:.4f}, N = {N}, k = {k}')
print('=' * 65)

for gamma in gammas:
    q = q_table[gamma]
    left = S_E * (1 - q)
    right = S_E * (1 + q)

    print(f'\nγ = {gamma}:')
    print(f'  q = {q}')
    print(f'  Доверительный интервал: (S(1-q); S(1+q)) = ({left:.4f}; {right:.4f})')

Параметр: nu (кг/см³)
S = 0.0543, N = 109, k = 108

γ = 0.95:
  q = 0.143 
  Доверительный интервал: (S(1-q); S(1+q)) = (0.0465; 0.0620)

γ = 0.99:
  q = 0.198 
  Доверительный интервал: (S(1-q); S(1+q)) = (0.0435; 0.0650)

Параметр: E (кг/см²)
S = 22.3464, N = 109, k = 108

γ = 0.95:
  q = 0.143
  Доверительный интервал: (S(1-q); S(1+q)) = (19.1509; 25.5419)

γ = 0.99:
  q = 0.198
  Доверительный интервал: (S(1-q); S(1+q)) = (17.9218; 26.7710)


Доверительные интервалы для СКО построены по формуле $(S(1-q);\ S(1+q))$. Значения $q$ взяты из Приложения 7 для $n = 100$ (ближайшее к $N = 109$): $q = 0.143$ при $\gamma = 0.95$ и $q = 0.198$ при $\gamma = 0.99$.

Параметр $q$ одинаков для обоих признаков, так как зависит только от $\gamma$ и $N$. При увеличении надёжности с $\gamma = 0.95$ до $\gamma = 0.99$ значение $q$ возрастает (с 0.143 до 0.198), что приводит к расширению доверительного интервала — закономерность аналогична пункту 1.

3. Проверить гипотезу о нормальности заданного распределения с помощью критерия $\chi^2$ (Пирсона). Для этого необходимо найти теоретические частоты и вычислить наблюдаемое значение критерия. Для удобства вычисления необходимо заполнить табл. 1.

In [28]:
from scipy.stats import norm

num_intervals = round(1 + 3.322 * np.log10(N))
print(f'Число интервалов (Стёрджес): K = {num_intervals}')


def laplace_func(val):
    if val == -np.inf:
        return -0.5
    if val == np.inf:
        return 0.5
    return norm.cdf(val) - 0.5


def pearson_test(sample, mean_val, std_val, size, num_bins, feature):
    edges = np.linspace(sample.min(), sample.max(), num_bins + 1)

    # стандартизация: z_i = (x_i - x_mean) / S, z_1 = -inf, z_{K+1} = +inf
    z_bounds = np.array([(edges[i] - mean_val) / std_val for i in range(num_bins + 1)])
    z_bounds[0], z_bounds[-1] = -np.inf, np.inf

    # вероятности через функцию Лапласа
    probs = np.array([laplace_func(z_bounds[i + 1]) - laplace_func(z_bounds[i])
                       for i in range(num_bins)])

    # теоретические и наблюденные частоты
    expected = size * probs
    observed = np.histogram(sample, bins=edges)[0].astype(float)

    # хи-квадрат
    residuals = (observed - expected) ** 2 / expected
    chi2_result = residuals.sum()

    # подписи интервалов
    labels = []
    for i in range(num_bins):
        lo = '(-inf' if i == 0 else f'[{edges[i]:.4f}'
        hi = '+inf)' if i == num_bins - 1 else f'{edges[i + 1]:.4f})'
        labels.append(f'{lo}, {hi}')

    # датафрейм
    rows = {
        'i': list(range(1, num_bins + 1)) + ['sum'],
        '[xi, xi+1)': labels + [''],
        'ni': list(observed.astype(int)) + [int(observed.sum())],
        'pi': list(np.round(probs, 4)) + [round(probs.sum(), 4)],
        "n'i": list(np.round(expected, 4)) + [round(expected.sum(), 4)],
        "(ni - n'i)^2": list(np.round((observed - expected) ** 2, 4)) + ['-'],
        "(ni-n'i)^2/n'i": list(np.round(residuals, 4)) + [round(chi2_result, 4)],
        'ni^2': list((observed ** 2).astype(int)) + ['-'],
        "ni^2/n'i": list(np.round(observed ** 2 / expected, 4)) + [round((observed ** 2 / expected).sum(), 4)],
    }
    result_table = pd.DataFrame(rows)

    print(f'Признак: {feature}')
    print(f'x_mean = {mean_val:.4f}, S = {std_val:.4f}, N = {size}, K = {num_bins}')
    print(f'Хи-квадрат наблюдаемое = {chi2_result:.4f}')

    # критическое значение
    deg_free = num_bins - 3
    chi2_crit = stats.chi2.ppf(1 - 0.05, df=deg_free)

    print(f'df = K - 3 = {num_bins} - 3 = {deg_free}')
    print(f'Хи-квадрат критическое = {chi2_crit:.4f}')

    if chi2_result <= chi2_crit:
        print(f'{chi2_result:.4f} <= {chi2_crit:.4f} -> H0 принимается\n')
    else:
        print(f'{chi2_result:.4f} > {chi2_crit:.4f} -> H0 отвергается\n')

    return observed, expected, result_table

Число интервалов (Стёрджес): K = 8


In [29]:
obs_nu, exp_nu, tbl_nu = pearson_test(nu.values, x_mean_nu, S_nu, N, num_intervals, 'nu (kg/cm^3)')
tbl_nu

Признак: nu (kg/cm^3)
x_mean = 0.4496, S = 0.0543, N = 109, K = 8
Хи-квадрат наблюдаемое = 5.0916
df = K - 3 = 8 - 3 = 5
Хи-квадрат критическое = 11.0705
5.0916 <= 11.0705 -> H0 принимается



,i,"[xi, xi+1)",ni,pi,n'i,(ni - n'i)^2,(ni-n'i)^2/n'i,ni^2,ni^2/n'i
0,1,"(-inf, 0.3541)",6,0.0393,4.2882,2.9304,0.6834,36,8.3952
1,2,"[0.3541, 0.3882)",7,0.0900,9.8059,7.8728,0.8029,49,4.9970
2,3,"[0.3882, 0.4224)",23,0.1789,19.4983,12.2622,0.6289,529,27.1306
3,4,"[0.4224, 0.4565)",23,0.2426,26.4455,11.8712,0.4489,529,20.0034
4,5,"[0.4565, 0.4906)",24,0.2245,24.4691,0.2201,0.0090,576,23.5399
5,6,"[0.4906, 0.5248)",15,0.1417,15.4447,0.1978,0.0128,225,14.5681
6,7,"[0.5248, 0.5589)",10,0.0610,6.6487,11.2314,1.6893,100,15.0406
7,8,"[0.5589, +inf)",1,0.0220,2.3998,1.9594,0.8165,1,0.4167
8,sum,,109,1.0000,109.0000,-,5.0916,-,114.0916


In [30]:
obs_e, exp_e, tbl_e = pearson_test(E.values, x_mean_E, S_E, N, num_intervals, 'E (kg/cm^2)')
tbl_e

Признак: E (kg/cm^2)
x_mean = 127.9128, S = 22.3464, N = 109, K = 8
Хи-квадрат наблюдаемое = 7.6029
df = K - 3 = 8 - 3 = 5
Хи-квадрат критическое = 11.0705
7.6029 <= 11.0705 -> H0 принимается



,i,"[xi, xi+1)",ni,pi,n'i,(ni - n'i)^2,(ni-n'i)^2/n'i,ni^2,ni^2/n'i
0,1,"(-inf, 85.6375)",7,0.0293,3.1891,14.5231,4.5540,49,15.3649
1,2,"[85.6375, 100.1750)",5,0.0780,8.5016,12.261,1.4422,25,2.9406
2,3,"[100.1750, 114.7125)",15,0.1701,18.5411,12.5397,0.6763,225,12.1352
3,4,"[114.7125, 129.2500)",30,0.2465,26.8687,9.8052,0.3649,900,33.4963
4,5,"[129.2500, 143.7875)",24,0.2374,25.8778,3.5262,0.1363,576,22.2584
5,6,"[143.7875, 158.3250)",19,0.1520,16.5642,5.9331,0.3582,361,21.7940
6,7,"[158.3250, 172.8625)",7,0.0646,7.0446,0.002,0.0003,49,6.9557
7,8,"[172.8625, +inf)",2,0.0221,2.4129,0.1705,0.0707,4,1.6578
8,sum,,109,1.0000,109.0000,-,7.6029,-,116.6029


4. Доказать, что
$$\chi^2_{\text{набл}} = \sum_{i=1}^{k} \frac{n_i^2}{n'_i} - n.$$
Проконтролировать корректность вычисления $\chi^2_{\text{набл}}$.

Раскроем скобки в формуле $\chi^2_{\text{набл}}$:

$$\chi^2_{\text{набл}} = \sum_{i=1}^{K} \frac{(n_i - n'_i)^2}{n'_i} = \sum_{i=1}^{K} \frac{n_i^2 - 2n_i n'_i + n'^2_i}{n'_i} = \sum_{i=1}^{K}\left(\frac{n_i^2}{n'_i} - 2n_i + n'_i\right)$$

$$= \sum_{i=1}^{K}\frac{n_i^2}{n'_i} - 2\sum_{i=1}^{K}n_i + \sum_{i=1}^{K}n'_i$$

Так как $\sum_{i=1}^{K} n_i = N$ и $\sum_{i=1}^{K} n'_i = N$, получаем:

$$\chi^2_{\text{набл}} = \sum_{i=1}^{K}\frac{n_i^2}{n'_i} - 2N + N = \sum_{i=1}^{K}\frac{n_i^2}{n'_i} - N$$

Что и требовалось доказать.

In [31]:
for tag, obs, exp in [('nu', obs_nu, exp_nu), ('E', obs_e, exp_e)]:
    formula_1 = np.sum((obs - exp) ** 2 / exp)
    formula_2 = np.sum(obs ** 2 / exp) - N

    print(f'Признак {tag}:')
    print(f'  Основная формула:        {formula_1:.4f}')
    print(f'  Преобразованная формула:  {formula_2:.4f}')
    print(f'  Совпадение: {np.isclose(formula_1, formula_2)}')
    print()

Признак nu:
  Основная формула:        5.0916
  Преобразованная формула:  5.0916
  Совпадение: True

Признак E:
  Основная формула:        7.6029
  Преобразованная формула:  7.6029
  Совпадение: True



Для обоих признаков результаты вычисления $\chi^2_{\text{набл}}$ по основной и преобразованной формулам совпадают: для $\nu$ — $5.0916 = 5.0916$, для $E$ — $7.6029 = 7.6029$. Это подтверждает корректность вычислений и справедливость доказанного тождества $\chi^2_{\text{набл}} = \sum \frac{n_i^2}{n'_i} - N$.

5. По заданному уровню значимости $\alpha = 0.05$ и числу степеней свободы $df$ найти критическую точку $\chi^2_{\text{крит}}$ и сравнить с наблюдаемым значением. Сделать выводы.

По заданному уровню значимости $\alpha = 0.05$ и числу степеней свободы $df = K - 3$ находим критическую точку $\chi^2_{\text{крит}}$ и сравниваем с наблюдаемым значением. Если $\chi^2_{\text{набл}} \leq \chi^2_{\text{крит}}$ — гипотеза $H_0$ принимается, иначе — отвергается.

In [32]:
alpha = 0.05
for tag, obs, exp in [('nu', obs_nu, exp_nu), ('E', obs_e, exp_e)]:
    chi2_obs = np.sum((obs - exp) ** 2 / exp)
    deg_free = len(obs) - 3
    chi2_crit = stats.chi2.ppf(1 - alpha, df=deg_free)

    print(f'Признак {tag}:')
    print(f'  alpha = {alpha}, df = K - 3 = {len(obs)} - 3 = {deg_free}')
    print(f'  Хи-квадрат наблюдаемое  = {chi2_obs:.4f}')
    print(f'  Хи-квадрат критическое  = {chi2_crit:.4f}')

    if chi2_obs <= chi2_crit:
        print(f'  {chi2_obs:.4f} <= {chi2_crit:.4f} -> H0 принимается')
    else:
        print(f'  {chi2_obs:.4f} > {chi2_crit:.4f} -> H0 отвергается')
    print()

Признак nu:
  alpha = 0.05, df = K - 3 = 8 - 3 = 5
  Хи-квадрат наблюдаемое  = 5.0916
  Хи-квадрат критическое  = 11.0705
  5.0916 <= 11.0705 -> H0 принимается

Признак E:
  alpha = 0.05, df = K - 3 = 8 - 3 = 5
  Хи-квадрат наблюдаемое  = 7.6029
  Хи-квадрат критическое  = 11.0705
  7.6029 <= 11.0705 -> H0 принимается



Для обоих признаков при уровне значимости $\alpha = 0.05$ и числе степеней свободы $df = K - 3 = 5$ наблюдаемые значения не превышают критическое: для $\nu$ — $5.09 \leq 11.07$, для $E$ — $7.60 \leq 11.07$. Гипотеза $H_0$ о нормальном распределении принимается для обоих признаков. Расхождения между эмпирическими и теоретическими частотами статистически незначимы.

## Выводы

В рамках практической работы были построены интервальные оценки параметров распределения для двух признаков ($\nu$ и $E$) и проведена проверка гипотезы о нормальности.

Доверительные интервалы для математического ожидания получены на основе распределения Стьюдента ($k = N - 1 = 108$): для $\nu$ — $(0.4393; 0.4599)$ при $\gamma = 0.95$ и $(0.4359; 0.4632)$ при $\gamma = 0.99$; для $E$ — $(123.67; 132.16)$ и $(122.30; 133.53)$ соответственно. Доверительные интервалы для СКО построены по формуле $(S(1-q);\ S(1+q))$ с табличными значениями $q$ (Приложение 7): для $\nu$ — $(0.0465; 0.0620)$ при $\gamma = 0.95$ и $(0.0435; 0.0650)$ при $\gamma = 0.99$; для $E$ — $(19.15; 25.54)$ и $(17.92; 26.77)$. В обоих случаях увеличение надёжности приводит к расширению интервалов при фиксированном объёме выборки.

Проверка гипотезы $H_0$ о нормальном распределении критерием Пирсона при $\alpha = 0.05$ и $df = 5$ показала: $\chi^2_{\text{набл}} = 5.09$ для $\nu$ и $\chi^2_{\text{набл}} = 7.60$ для $E$, что не превышает $\chi^2_{\text{крит}} = 11.07$. Гипотеза не отвергается для обоих признаков. Эквивалентность формул $\chi^2_{\text{набл}} = \sum \frac{(n_i - n'_i)^2}{n'_i} = \sum \frac{n_i^2}{n'_i} - N$ подтверждена.

Принятие гипотезы о нормальности обосновывает корректность применённых методов интервального оценивания, основанных на распределениях Стьюдента и $\chi^2$.